10_cdisc_questions.md

```diff
@@ -0,0 +1,68 @@
+1. What is the main focus of Good Clinical Practices (GCP)? [comment]: <> (This is not appropriate for CDISC section, perhaps could be moved general programming section)

```

Comment:
We agreed to move this question out of CDISC section. More appropriate questions perhaps should be: 
1. What are CDISC standards? 
2. What is CDASH standard
3. What is SDTM standard
4. What is ADam Standard
5. What is Controlled terminology
6. SDTM Model and IG
7. ADaM Model & IG

Reply:
Can we add some of these to the MD file? I accepted the merge